src/Console/Deployer.php

```diff
@@ -81,7 +81,7 @@ public function deploy(SymfonyStyle $io, bool $dryRun, ?string $stage): void
             if ($stage !== null) {
                 $serverlessCommand .= ' --stage ' . escapeshellarg($stage);
             }
-            $process = new Process($serverlessCommand, '.bref/output');
+            $process = new Process([$serverlessCommand, '.bref/output']);
```

Comment:
I guess this should be

```suggestion
            $process = new Process([$serverlessCommand] '.bref/output');
```